src/elements/header/header-button/header-button.visual.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffStandardStates } from '../../core/testing/private.js';
+
+import './header-button.js';
+
+describe(`sbb-header-button`, () => {
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    for (const state of visualDiffStandardStates) {
+      it(
+        state.name,
+        state.with(async (setup) => {
+          await setup.withFixture(html`
+            <sbb-header-button icon-name="hamburger-menu-small" type="button"
+              >Menu</sbb-header-button
+            >
+          `);
+        }),
+      );
+
+      it(
+        `multiple=true ${state.name}`,
```

Comment:
Does the multiple add any value?
I don't see any CSS rule in the header that affects multiple elements

Reply:
I'd vote to remove them. multiple are tested with sbb-header?